release/nrc/str.sencoten/source/saanich.tsv

```diff
@@ -1,8 +1,7 @@
 TŦE	13644

```

Comment:
I'm a little puzzled by this -- the only change I can see is a fairly minor change in frequency and the removal of one common word `ȻNES`?

Reply:
I've added a few words based on my interaction with Dr. Montler. I initially rejected all words with an "F", because "F" is not a native phoneme in SENĆOŦEN; however, I learned that it _is_ valid in loan words. He also cleaned up the tokenization of the original list, giving slightly better results.

"ȻNES"  is still there; it has just dropped significantly lower in the list. As written in the comments of `fix-wordlist.py`, "ȻNES" is not an orthographically correct word, but rather a common token typed by older L2 learners. Kind of like "would of" "could of" in English, in that it's incorrect, but it's widely used enough that it'd be annoying if it _wasn't_ there. So I let it be in the list with a relative frequency lower than "Ȼ" and "NE" which _are_ orthographically correct.